src/hooks/useForm.js

```diff
@@ -62,6 +63,11 @@ function useForm ({
     setInput(newInput)
   }
 
+  function handleRadioChange (e) {
```

Comment:
We don't want to add a custom handler for radios, the default `handleChange` does the same thing. Adding a separate state object for different form inputs will make this hook impractical in the long term. You can simply observe the `input` state that is returned from the hook instead of creating a new state object.